server/test/api/employee.js

```diff
@@ -68,7 +68,7 @@ describe('The /employees API endpoint :: ', function () {
       location_id : 'ffe563ef-781c-4551-a080-7cec135351ff'
   };
 
-  it('POST /employee should create a new employee', function () {
+  it('METHOD : POST PATH : /employee,  should create a new employee', function () {
```

Comment:
Do we really need to write "METHOD" and "PATH"?  Is there any confusion about what these might be?